dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Validation31TestServlet.java

```diff
@@ -80,6 +96,12 @@ public void nestedRecordsTest() throws Exception {
     public void convertGroupsRecordsTest() throws Exception {
         // TODO See section 5.4 of the Bean Validation specification for groups
         // and section 5.4.5 for group conversion
+        Registeration reg = new Registeration("x1asas", false);
+        Company cmp2 = new Company("CompanyName1", reg);
+        Set<ConstraintViolation<Company>> constraintViolations = validator.validate(cmp2);
+        assertTrue("Record Person(null) should have validated with one violation", constraintViolations.size() > 0);
+        System.out.println(constraintViolations.size());
```

Comment:
These two lines can both be removed

Reply:
I have updated the class by removing those 2 lines